routers/api/v1/repo/release.go

```diff
@@ -55,6 +56,18 @@ func GetRelease(ctx *context.APIContext) {
 	ctx.JSON(200, release.APIFormat())
 }
 
+func getPagesInfo(ctx *context.APIContext) (int, int) {
+	page := ctx.QueryInt("page")
+	if page == 0 {
+		page = 1
+	}
+	perPage := ctx.QueryInt("per_page")
```

Comment:
It should not allow `perPage` to be more then `MaxResponseItems`

Reply:
done.